src/container.cpp

```diff
@@ -256,7 +254,7 @@ ReturnValue Container::queryAdd(int32_t index, const Thing& thing, uint32_t coun
 		return RETURNVALUE_NOTPOSSIBLE;
 	}
 
-	const Item* item = thing.getItem();
+	auto item = thing->getItem();
```

Comment:
Could this be `const auto`
```suggestion
	const auto item = thing->getItem();
```